tests/server/services/test_scheduler.py

```diff
@@ -124,6 +124,48 @@ async def test_create_schedules_from_deployment(
     )
 
 
+async def test_create_parametrized_schedules_from_deployment(
+    flow, session, simple_parameter_schema
+):
+    schedule = schemas.schedules.IntervalSchedule(
+        interval=datetime.timedelta(days=30),
+        anchor_date=pendulum.now("UTC"),
+    )
+
+    await models.deployments.create_deployment(
+        session=session,
+        deployment=schemas.core.Deployment(
+            name="test",
+            flow_id=flow.id,
+            parameter_openapi_schema=simple_parameter_schema.model_dump_for_openapi(),
+            schedules=[
+                schemas.core.DeploymentSchedule(
+                    schedule=schedule,
+                    parameters={"name": "whoami"},
+                    active=True,
+                ),
+            ],
+        ),
+    )
```

Comment:
It might be good to include a parameter on the deployment to ensure that the schedule parameter takes precedence.